.github/workflows/verify.yaml

```diff
@@ -25,6 +19,26 @@ jobs:
           checkstyle_version: '9.0'
           reporter: 'github-check'
           filter_mode: 'nofilter'
+          properties_file: resources/edc-checkstyle-ignore-enhanced.properties
+
+      - name: Run Checkstyle with additional checks for changed lines
+        uses: nikitasavinov/checkstyle-action@0.4.0
+        with:
+          checkstyle_config: resources/edc-checkstyle-config.xml
+          level: error
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          tool_name: 'checkstyle-enhanced'
+          checkstyle_version: '9.0'
+          # Filtering does not work on github-check, needs github-pr-check
+          reporter: 'github-pr-check'
+          # Include only violations on added or modified files
+          filter_mode: 'file'
```

Comment:
here we do not need to specify `properties_file` ? similar to the above check.

Reply:
No, we run with defaults

Reply:
Removed